packages/odyssey-storybook/src/components/odyssey-mui/Form/Form.stories.tsx

```diff
@@ -185,23 +188,8 @@ export default storybookMeta;
 
 const Template: StoryObj<FormProps> = {
   render: function (args) {
-    return (
-      <Form
-        title={args.title}
-        name={args.name}
-        description={args.description}
-        formActions={args.formActions}
-        alert={args.alert}
-        autoCompleteType={args.autoCompleteType}
-        encodingType={args.encodingType}
-        method={args.method}
-        noValidate={args.noValidate}
-        target={args.target}
-        id={args.id}
-      >
-        {args.children}
-      </Form>
-    );
+    const { children } = args;
+    return <Form {...args}>{children}</Form>;
```

Comment:
This is unnecessary. `children` can be passed as a prop, and it will happen automatically. The whole `render` function can go.